tests/osaka/eip7692_eof_v1/eip7873_tx_create/test_creation_tx.py

```diff
@@ -0,0 +1,86 @@
+"""Test bad TXCREATE cases."""
+
+import pytest
+
+from ethereum_test_exceptions.exceptions import TransactionException
+from ethereum_test_tools import (
+    Account,
+    Alloc,
+    Environment,
+    StateTestFiller,
+    Transaction,
+)
+from ethereum_test_tools.code.generators import Initcode as LegacyInitcode
+
+from .. import EOF_FORK_NAME
+from ..eip7620_eof_create.helpers import (
+    smallest_initcode_subcontainer,
+    smallest_runtime_subcontainer,
+)
+
+REFERENCE_SPEC_GIT_PATH = "EIPS/eip-7873.md"
+REFERENCE_SPEC_VERSION = "1115fe6110fcc0efc823fb7f8f5cd86c42173efe"
+
+pytestmark = pytest.mark.valid_from(EOF_FORK_NAME)
+
+
```

Comment:
```suggestion
@pytest.mark.with_all_contract_creating_tx_types()
```
We should add this marker and parameter `tx_type` to repeat this same test with all current and future transaction types that can create contracts.

If there's a tx type that needs to be skipped which does not create a contract, you can use the selector: https://eest.ethereum.org/main/writing_tests/test_markers/#selector